src/templates/work.development.html

```diff
@@ -12,7 +12,6 @@
   <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
   <link rel="stylesheet" href="https://static.beenest.com/theme/theme.min.css">
   <base href="/" />
-  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyB7DfwQwnhYjPzx8UIF0JHlgVeNwSDnZkY" async defer></script>
```

Comment:
removing because we don't need this on the product site, only admin

Reply:
dont we need for the autocomplete?

Reply:
we have it in the google search container component

Reply:
we had this in the admin for the maps when creating a listing, but on our product site we have them as wrappers instead of globally

Reply:
+1